src/styles/utils/index.ts

```diff
@@ -499,6 +499,10 @@ function getButtonStyleWithIcon(
             return useDefaultButtonStyles ? styles.buttonLarge : {...styles.buttonLarge, ...(hasText ? verticalStyle : styles.ph0)};
         }
         default: {
+            if (hasIcon && !hasText) {
+                return {...styles.buttonMedium, ...styles.ph0};
+            }
```

Comment:
What case is this fixing?

Reply:
@s77rt this one https://github.com/Expensify/App/pull/38275#issuecomment-1997512298